src/pages/Experiment.less

```diff
@@ -72,3 +98,142 @@
         font-size: 48px;
     }
 }
+
+.btn {
+    text-transform: uppercase;
+    font-size: 15px;
+    font-weight: 300;
+    letter-spacing: 2px;
+    padding-right: 22px;
+    padding-left: 22px;    
+}
+
+.btn, .btn-lg, .btn-group-lg > .btn {
+    border-radius: 0;
+}
+
+.btn-success {
+    color: #004c2a;
+    background-color: #66ffba;
+    border: none;
+}
+
+.btn-danger {
+    color: #61312c;
+    background-color: #f37b6f;
+    border: none;
+}
+
+.btn-default {
+    color: #191b1c;
+    background-color: #808b8c;
+    border: none;
+}
+
+.btn-default:hover,
+.btn-danger:hover,
+.btn-success:hover,
+.btn-info:hover,
+.btn-default:focus,
+.btn-danger:focus,
+.btn-success:focus,
+.btn-info:focus {
+    color: #d8dcdc;
+    background-color: #0c0d0e;    
+}
+
+.btn-success:hover,
+.btn-success:focus {
+    color: #66ffba;
+}
+.btn-danger:hover,
+.btn-danger:focus {
+    color: #f37b6f;
+}
+
+.btn-info {
+    color: #006166;
+    background-color: #00f3ff;
+    border: none;
+    padding-top: 10px;
+    padding-bottom: 10px
+}
+
+.btn-info:hover,
+.btn-info:focus {
+    color: #00f3ff;
+}
+
+.navbar .container {
+    width: 100%;
+}
+
+.navbar > .container .navbar-brand, .navbar > .container-fluid .navbar-brand {
+    margin-left: 42px;
+}
+.navbar-header h1.navbar-brand {
+    margin: 0 0 0 42px;
+    padding: 14px 0 0 0;
+
+    a {
+        color: #808b8c;
+        font-size: 12px;
+        text-transform: uppercase;
+        letter-spacing: 1px;
+        font-weight: 300;
+        text-decoration: none;
+
+        img {
+            width: 30px;
+            margin-right: 10px;
+            vertical-align: -4px;
+        }
+    }
+
+    a:hover {
+        text-decoration: none;
+        color: #fff;
+    }
+}
+
+.navbar-right {
+    margin-right: 28px;
+}
+
+.navbar-default .navbar-nav > li > a {
+    color: #808b8c;
+    font-size: 12px;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    font-weight: 300;
+    text-decoration: none;
+}       
+
+.navbar-default .navbar-nav > li > a:hover {
+    text-decoration: none;
+    color: #fff;    
+}
+
+.progress {
+    background: #666a6a;
+    border-radius: 0;
+}
+
+.progress-bar {
+    background: #a0a2a2; 
+    background-image: none;
+}
+
+.text-right .btn.btn-default {
+    
+}
+
+.ex-progress__percent {
+    color: #a0a2a2;
+    text-transform: uppercase;
+    font-size: 13px;
+    font-weight: 500;
+    letter-spacing: 1px;
+    padding-right: 8px;
+    padding-top: 1px;
+}
```

Comment:
missing `new line`;
all files should finish with (only one) line break.

Reply:
Thanks @dpordomingo ! Will iterate, commit and push.